usaspending_api/disaster/management/sql/disaster_covid19_file_a.sql

```diff
@@ -0,0 +1,81 @@
+    FROM "dabs_submission_window_schedule"
+    WHERE
+        "dabs_submission_window_schedule"."submission_reveal_date" <= now()
+    GROUP BY
+        "dabs_submission_window_schedule"."submission_fiscal_year",
+        "dabs_submission_window_schedule"."is_quarter"
+)
+
+SELECT
+    agency."name" AS "owning_agency_name",
+    CONCAT('FY', gtas."fiscal_year", 'P', gtas."fiscal_period") AS "submission_period",
+    COALESCE(taa."allocation_transfer_agency_id",
+        CASE WHEN array_upper(string_to_array(gtas."tas_rendering_label", '-'), 1) = 5
+        THEN SPLIT_PART(gtas."tas_rendering_label", '-', 1)
+        ELSE NULL END
+    ) AS "allocation_transfer_agency_identifer_code",
+    COALESCE(taa."agency_id",
+        CASE WHEN array_upper(string_to_array(gtas."tas_rendering_label", '-'), 1) = 5
+        THEN SPLIT_PART(gtas."tas_rendering_label", '-', 2)
+        ELSE SPLIT_PART(gtas."tas_rendering_label", '-', 1) END
+    ) AS "agency_identifier_code",
+    COALESCE(taa."beginning_period_of_availability",
+        CASE WHEN SPLIT_PART(REVERSE(gtas."tas_rendering_label"), '-', 3) != 'X'
+        THEN REVERSE(SPLIT_PART(SPLIT_PART(REVERSE(gtas."tas_rendering_label"), '-', 3), '/', 2))
+        ELSE NULL END
+    ) AS "beginning_period_of_availability",
+    COALESCE(taa."ending_period_of_availability",
+        CASE WHEN SPLIT_PART(REVERSE(gtas."tas_rendering_label"), '-', 3) = 'X'
+        THEN REVERSE(SPLIT_PART(SPLIT_PART(REVERSE(gtas."tas_rendering_label"), '-', 3), '/', 1))
+        ELSE NULL END
+    ) AS "ending_period_of_availability",
+    COALESCE(taa."availability_type_code",
+        CASE WHEN SPLIT_PART(REVERSE(gtas."tas_rendering_label"), '-', 3) = 'X'
+        THEN 'X'
+        ELSE NULL END
+    ) AS "availability_type_code",
+    COALESCE(taa."main_account_code", REVERSE(SPLIT_PART(REVERSE(gtas."tas_rendering_label"), '-', 2))) AS main_account_code,
+    COALESCE(taa."sub_account_code", REVERSE(SPLIT_PART(REVERSE(gtas."tas_rendering_label"), '-', 1))) AS sub_account_code,
+    gtas."tas_rendering_label" AS "treasury_account_symbol",
+    taa."account_title" AS "treasury_account_name",
+    (SELECT "name" FROM "toptier_agency" WHERE "toptier_agency"."toptier_code" = (
+        COALESCE(
+            taa."agency_id",
+                CASE WHEN array_upper(string_to_array(gtas."tas_rendering_label", '-'), 1) = 5
+                THEN SPLIT_PART(gtas."tas_rendering_label", '-', 2)
+                ELSE SPLIT_PART(gtas."tas_rendering_label", '-', 1) END
+        )
+    )) AS "agency_identifier_name",
+    (SELECT "name" FROM "toptier_agency" WHERE "toptier_agency"."toptier_code" = (
+    COALESCE(taa."allocation_transfer_agency_id",
+        CASE WHEN array_upper(string_to_array(gtas."tas_rendering_label", '-'), 1) = 5
+        THEN SPLIT_PART(gtas."tas_rendering_label", '-', 1)
+        ELSE NULL END
+        )
+    )) AS "allocation_transfer_agency_identifer_name",
```

Comment:
This should be pulling from the CGAC table.  ATA is a CGAC.

Reply:
Fixed